host-configs/llnl/toss_3_x86_64_ib/intel-oneapi@2021.3-c++17.cmake

```diff
@@ -0,0 +1,60 @@
+# Copyright (c) 2017-2021, Lawrence Livermore National Security, LLC and
+# other BLT Project Developers. See the top-level LICENSE file for details
+# 
+# SPDX-License-Identifier: (BSD-3-Clause)
+
+#------------------------------------------------------------------------------
+# Example host-config file for the quartz cluster at LLNL
+#------------------------------------------------------------------------------
+#
+# This file provides CMake with paths / details for:
+#  C,C++, & Fortran compilers + MPI
+# 
+#------------------------------------------------------------------------------
+
+#------------------------------------------------------------------------------
+# intel-oneapi.2021.3 compilers
+#------------------------------------------------------------------------------
+
+set(INTELLLVM_VERSION "intel-oneapi.2021.3")
+set(INTELLLVM_HOME "/usr/tce/packages/intel/${INTELLLVM_VERSION}")
+
+# c compiler
+set(CMAKE_C_COMPILER "${INTELLLVM_HOME}/bin/icx" CACHE PATH "")
+
+# cpp compiler
+set(CMAKE_CXX_COMPILER "${INTELLLVM_HOME}/bin/icpx" CACHE PATH "")
+
+# fortran support
+set(ENABLE_FORTRAN ON CACHE BOOL "")
+
+# fortran compiler
+set(CMAKE_Fortran_COMPILER "${INTELLLVM_HOME}/bin/ifx" CACHE PATH "")
+
+#------------------------------------------------------------------------------
+# Extra flags
+#------------------------------------------------------------------------------
+
+set(BLT_CXX_STD "c++17")
+
+# Use non-system default stdlib
+set(GCC_VERSION "gcc-9.3.1")
+set(GCC_HOME "/usr/tce/packages/gcc/${GCC_VERSION}/rh")
```

Comment:
Please add a comment that gcc versions `8.3.1` and above require the `rh` suffix in their path

Reply:
What the difference?  There's so many links in the directories it's hard to see what's going on.

Reply:
From LC:

"The version up to and include 8.1.0 are standalone installations, while 8.3.1 and above actually point the build in /opt. Sorry for any confusion."

Reply:
OK, I remember. I ran into this problem earlier where scipy wanted a symbol from stdlibc++ but it could not be found.  scipy was compiled with 4.9.3 which had a newer version of stdlibc++ than the version 8.3.1 which wanted to use from /usr/lib64.  LC told me to load with the stdlibc++ from 8.1.0.  But if I use the rh directory, things should just work? I found a stdlibc++ directory under rh.